tests/integration/src/setup.rs

```diff
@@ -237,3 +237,8 @@ fn last_events(n: usize) -> Vec<RuntimeEvent> {
 pub fn expect_events(e: Vec<RuntimeEvent>) {
     assert_eq!(last_events(e.len()), e);
 }
+
+#[allow(dead_code)]
+pub fn init_env_logger() {
+    let _ = env_logger::builder().is_test(true).try_init();
+}
```

Comment:
Why is this needed?

Reply:
It's for `TRACE` / `DEBUG` logging setup in tests. It's not essential for tests to run but super useful for debugging. I think it's worth keeping it, otherwise need to add and remove it (and the deps in `Cargo.toml`) every time debugging.